app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import datetime
+import time
+
+
+def files_creator():
+    while True:
+        time_now = datetime.datetime.now()
+        time_now = "app-" +\
+                   str(time_now.hour) +\
+                   "_" + str(time_now.minute) +\
+                   "_" + str(time_now.second) +\
+                   ".log"
+
+        with open(time_now, "w") as f:
+            f.write(str(datetime.datetime.now()))
```

Comment:
Could you use the value you got in line 7?
To do this you need to store the file name in a separate variable, not in `time_now`